libs/genai/langchain_google_genai/embeddings.py

```diff
@@ -253,7 +253,10 @@ def embed_query(
         Returns:
             Embedding for the text.
         """
-        task_type = self.task_type or "RETRIEVAL_QUERY"
+        # Determine the task_type to use for this specific query
```

Comment:
nits: comment is not needed

you can also just do ` task_type if task_type else self.task_type`, no need for is not None